share/packages/mbgft.xml

```diff
@@ -17,6 +17,9 @@
 	    <tasks>singlets,triplets</tasks> <!-- default task is perturbative QP only, other options: qpdiag, singlets, triplets, all -->
 	    <store></store>
         <exctotal>25</exctotal>
+        <davidson>1</davidson>
```

Comment:
this should be a suboption 

<davidson>
   <jocc></jocc>
   <jsolve></jsolve>
</davidson>

The names are also not very clean. Better names please